### PR TITLE
Resolves Issue #15 - Add listLanguages method

### PR DESCRIPTION
**Pull request by mearns**

### CODE REVIEW

File: lib/core.js [hunk 1]
```diff
@@ -126,6 +127,11 @@ function registerLanguage(name, syntax) {
   }
 }
 
+/* Get a list of all registered languages */
+function listLanguages() {
+  return languageNames.slice(0, languageNames.length)
```
Comment (wooorm) [inline lib/core.js]:
```suggestion
  return languageNames.concat()
```

As you’re making a clone here, I think `concat` is a cleaner way to do it!

**Comment by mearns** [comment]
Awesome, thanks for the quick response and suggestions @wooorm ! I've pushed changes as requested (in 0e3ede7, c8f1957, and 36d9ab5). I'm very glad you indicated that I could just import the expected languages from highlight.js, I didn't realize that all the `registerLanguage` calls in index.js were autogenerated, this is so much better!

I'm not sure exactly how to do a `.includes` test, is this part of `tape`?

**Comment by wooorm** [comment]
@mearns Nice!

With includes, what I meant was something like this:

```js
t.ok(
  low.listLanguages().includes('--test-lang--'),
  'should include any additional languages that are registered'
)
```

It doesn’t make much of a difference, but it’s more in line with what you’re actually trying to test and what the message describes, rather than testing the previous assertion again as well!

**Comment by mearns** [comment]
(facepalm). Got it. Apparently I'm not awake enough this afternoon ;-). Changes pushed in 5bd515a81fc.

If/when you accept this PR, I'll plan to make a PR for refractor as well. 👍

**Pull request merged**